scripts/cleanup_datasets/rename_purged_datasets.py

```diff
@@ -8,7 +8,22 @@
 
 assert sys.version_info[:2] >= ( 2, 4 )
 
+def usage(prog) :
+    print "usage: %s file" % prog
+    print """
+Marks a set of files as purged and renames them. The input file should contain a
+list of files to be purged, one per line. The full path must be specified and
+must begin with /var/opt/galaxy.
```

Comment:
I would argue that they're probably not useful for most people. I have no clue why /var/opt/galaxy is hard-coded, though at least now people can know that without going through the code themselves. Personally I'd prefer if all of these scripts just parse the galaxy.ini for things like this, but that's probably a different PR.